Adding a full automation service call example

kozoke opened the pull request:
I did not realize I could add additional configurations by adding a second 'data' section.  May just be my ignorane but thought we could add a full example to help anyone else.

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Simple change to demonstrate the addtion of the second 'data' section to leverage additional android features.



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->


- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ X ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

kozoke committed:
Adding a full automation service call example

I did not realize I could add additional configurations by adding a second 'data' section.  May just be my ignorane but thought we could add a full example to help anyone else.

homeassistant commented:
Hi @kozoke, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20921).

Once you do that we will be able to review and accept this pull request.

Thanks!

frenck commented on source/_integrations/nfandroidtv.markdown:
```diff
@@ -125,4 +125,20 @@ icon:
+  - service: notify.living_room_tv
+    data:
+      title: "Thanks for the water!"
+      message: "Nigel is {{ states.sensor.nigel_moisture.state }}% moisture"
```
Don't access `states` directly, as it won't be able to handle occurrences when the entity isn't available/loaded yet.

Instead use the `states()` function

kozoke replied:
Understood.  Not quite sure how to edit it but I will try or resubmit a PR if it does not work.  (Still learning) :-)

kozoke replied:
Hi Frenck.  Big fan of your work.   I'm a bit new to using git etc and appreciate your patience.  
I agree with the changes/suggestions. I'm not sure how to implment the states() change - I have it fixed but unsure if I need cancel this PR (unlikely) or somehow edit this one?  I'll do some git research to address it.  thx.

frenck commented on source/_integrations/nfandroidtv.markdown:
````diff
@@ -125,4 +125,20 @@ icon:
+        transparency: "80%"
+        color: "teal"
+        interrupt: 0
+ ```yaml
````
You can remove `yaml` here

kozoke replied:
I'm not sure how this works 'you can remove yaml here' - I am clicking on th etext but I can't edit ithere.  I'll go back to the PR and try.  Thanks again for your patience.

frenck commented on source/_integrations/nfandroidtv.markdown:
````diff
@@ -125,4 +125,20 @@ icon:
   auth: "digest"
 ```
 
+Example of automation serivce call, full cofiguration:
````
```suggestion
Example of an automation with an service call, full configuration:
```

kozoke committed:
Update source/_integrations/nfandroidtv.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

kozoke commented on source/_integrations/nfandroidtv.markdown:
```diff
@@ -125,4 +125,20 @@ icon:
+  - service: notify.living_room_tv
+    data:
+      title: "Thanks for the water!"
+      message: "Nigel is {{ states.sensor.nigel_moisture.state }}% moisture"
```
- service: notify.living_room_tv
      data:
        title: "Thanks for the water!"
        message: Nigel is {{ states('sensor.nigel_moisture') }}% moisture
        data:
          duration: 4
          position: "bottom-left"
          fontsize: "medium"
          transparency: "80%"
          color: "teal"
          interrupt: 0

kozoke reviewed (commented):
Sorry Frenck, I cannot figure out how to make the change.  Do I cancel this pull request and make a new committ and pull request?

kozoke committed:
Modified states() format per Frenck's guidance

Modified service call example adjusting the states() format.   Fixes this well documented issue. :-)  Avoid using states.sensor.temperature.state, instead use states('sensor.temperature'). It is strongly advised to use the states(), is_state(), state_attr() and is_state_attr() as much as possible, to avoid errors and error message when the entity isn’t ready yet (e.g., during Home Assistant startup).

kozoke committed:
Merge branch 'home-assistant:current' into patch-1

kozoke commented:
Hi. I made the changes in my patch-1 branch and committed the change.  I'm not sure how to change this PR to include that change?  :-)

frenck commented on source/_integrations/nfandroidtv.markdown:
````diff
@@ -125,4 +125,20 @@ icon:
 
+Example of an automation with an service call, full configuration:
+
+```yaml
+  - service: notify.living_room_tv
+    data:
+      title: "Thanks for the water!"
+      message: Nigel is {{ states('sensor.nigel_moisture') }}% moisture
+      data:
+        duration: 4
+        position: "bottom-left"
+        fontsize: "medium"
+        transparency: "80%"
+        color: "teal"
+        interrupt: 0
+ ```yaml
````
````suggestion
{% raw %}

```yaml
service: notify.living_room_tv
data:
  title: "Thanks for the water!"
  message: "Nigel is {{ states('sensor.nigel_moisture') }}% moisture"
  data:
    duration: 4
    position: "bottom-left"
    fontsize: "medium"
    transparency: "80%"
    color: "teal"
    interrupt: 0
```

{% endraw %}
````

frenck committed:
Tweak

frenck reviewed (approved):
Thanks, @kozoke 👍
